deploy/scripts/src/test-accountManagement.ts

```diff
@@ -1373,25 +1491,46 @@ export function deleteAccount (): void {
 
       sleep(Math.random() * 3)
 
-      group('B04_DeleteAccount_07_APP_EnterAuthAppOTP POST', () => {
+      group('B04_DeleteAccount_07_01_APP_EnterAuthAppOTP POST', () => {
         const startTime = Date.now()
         res = http.post(env.signinURL + '/enter-authenticator-app-code',
           {
             _csrf: csrfToken,
             code: totp.generateTOTP()
           },
           {
-            tags: { name: 'B04_DeleteAccount_07_APP_EnterAuthAppOTP' }
+            tags: { name: 'B04_DeleteAccount_07_01_APP_EnterAuthAppOTP' }
           }
         )
         const endTime = Date.now()
 
+        check(res, {
+          'status is 200': (r) => r.status === 200,
+          'verify page content': (r) => (r.body as string).includes('Your services') || (r.body as string).includes('terms of use update')
+        })
+          ? transactionDuration.add(endTime - startTime)
+          : fail('Response Validation Failed')
+      })
+      group('B04_DeleteAccount_07_02_APP_AcceptTerms POST', () => {
+        const startTime = Date.now()
+        if ((res.body as string).includes('terms of use update')) {
```

Comment:
```suggestion
      if ((res.body as string).includes('terms of use update')) {
        group('B04_DeleteAccount_07_02_APP_AcceptTerms POST', () => {
        const startTime = Date.now()
```
Wrap the whole `group` in the `if` statement